### ISSUE TITLE
Images not working?

### ISSUE THREAD

**Issue by BLSK** [joshiejack/Enchiridion#79]
![2016-06-22_05 04 03](https://cloud.githubusercontent.com/assets/8443396/16260861/259bfe02-3837-11e6-80c8-c88875bb710c.png)

Thought I would add a nice image but then I get this... using the newest build
Enchiridion 1.9.4 3.1.5
Forge 1957

I tried an older build and still got it
using a png and a jpg both fail to show up.
not sure if its just not available just yet.
in case its a conflict:

Modpack List(so far)
[http://pastebin.com/mQwv1pa8](url)

**Comment by joshiejack** [comment]
Images work just fine for me, only thing looking back at code, what's your OS?

**Comment by BLSK** [comment]
Link does work... I had to copy and paste it in a new tab... its strange...
and I am using
Win10x64

**Comment by joshiejack** [comment]
Hmm, not what I thought it was then, I'll have to investigate later.

**Comment by BLSK** [comment]
Well I can tell you trying out the Infitech 2 which uses the 1.7.10 version of yours does work for me, butyeah I'll look back later on to see what you have to say... going to 'try' and see if its something on my end but so far nothing

**Comment by joshiejack** [comment]
Out of interest, the missing texture should end up getting printed to the log when it fails to load it, Can I get a copy of the log? Not that, that will help me much

**Comment by joshiejack** [comment]
Actually it will be better if you can: Use this debug version, load up the game, then try to insert some images then send me the full log. https://dl.dropboxusercontent.com/u/105284170/ShareX/2016/06/Enchiridion-1.9.4-3.1.5-debug.jar

**Comment by BLSK** [comment]
Here's where I get confused...
First image has a different png in the back and it rendered?

![2016-06-22_18 48 10](https://cloud.githubusercontent.com/assets/8443396/16286440/5142f3c6-38aa-11e6-94ce-959e4e458aee.png)
[logs.zip](https://github.com/joshiejack/Enchiridion/files/328904/logs.zip)

**Comment by BLSK** [comment]
OH MY GOD... my pc named the file wrong,  I just analyzed the Log and it says the image is a 'Name..png' putting in 2 dots. how... the simplest stupid thing... isn't that always the way it is...
As my first time looking at this mod...

one possible request:

I notice when I clicked on 'enable scroll the book doesn't like scretch out to fit the whole page and the background image(loving the things I am finding out :)) it also is the same issue...  I can't put in like a 2 times higher image so the page and background move together... image is static

Showing the splitting(image in background is just an test image):
2nd image is me figuring out goofup:

![2016-06-22_19 21 43](https://cloud.githubusercontent.com/assets/8443396/16287175/4695327c-38af-11e6-89e4-2d86413973ac.png)


![2016-06-22_19 18 28](https://cloud.githubusercontent.com/assets/8443396/16287031/4c1de99c-38ae-11e6-9262-9ddc306b971c.png)

**Comment by BLSK** [comment]
Ok person uses image of higher size for background:  I also can't revert it back to default background in case I choose to?

**Comment by BLSK** [comment]
Pro Tip:  Don't try to open a .jar file to search for an image in it... while adjusting background... crashes game cause it 'Trys to use the .jar as an image'  ... um... possible change stopping it from trying to pull up an invalid type?

**Comment by joshiejack** [comment]
Hehe. Always the silly things. I'm finding it kind of hard to follow what you're saying. Probably because it's late. But I think you're requesting the following things:

- A way to reset the background to default (without, editing json) (probably just a shift clicking the button?)
- I think you're asking for the option to make the background scroll as well, when scrolling is enabled?
- The background is a fixed default size, but you can edit that in the json, I doubt I'll add an ingame way for that
- As for the pro tip, not sure why you would do that anyway :P

**Comment by BLSK** [comment]
its early didn't have coffee... was thinking shortcut... fail XD
Yes to first 2... didn't see 'size changing in json but will look but yes I would like to see a scrolling background which would make the book look neater/cleaner and make us think your even cooler than you already are :)
